back-end/api/controllers/projects.controller.js

```diff
@@ -26,9 +26,37 @@ export default class ProjectsController {
      */
     static async apiDeleteProject(req, res, next) {
         try {
-            const projectName = req.query.project_name
+            const projectName = req.body.project_name
```

Comment:
currently, you are using the project_name to identify which project to delete. the code is working however i think it would be a good practice to not use name as 2 projects could have the same name but to use project id/ object id of the project as they are unique to the projects.